app/car.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import annotations
+
+
+class Car:
+    def __init__(self, brand: str, fuel_consumption: float) -> None:
+        self.brand = brand
+        self.fuel_consumption = fuel_consumption
+
+    @classmethod
+    def create_car(cls, car_info: dict[str, str | float | int]) -> Car:
+        return cls(
+            brand=car_info["brand"],
+            fuel_consumption=car_info["fuel_consumption"]
```

Comment:
Ensure that the keys 'brand' and 'fuel_consumption' are always present in the `car_info` dictionary to avoid KeyError. Consider using the `get` method with default values or handling exceptions.